pkg/testgridanalysis/testidentification/ocp_variants.go

```diff
@@ -160,6 +140,10 @@ var (
 		"release-openshift-ocp-installer-e2e-metal-compact-4.9",
 		"release-openshift-origin-installer-e2e-aws-sdn-network-stress-4.9",
 		"release-openshift-origin-installer-e2e-aws-sdn-network-stress-4.10",
+
+		// https://bugzilla.redhat.com/show_bug.cgi?id=2012844
+		"periodic-ci-openshift-release-master-nightly-4.10-e2e-metal-assisted",
+		"periodic-ci-openshift-release-master-nightly-4.10-e2e-metal-assisted-ipv6",
```

Comment:
This is a shame but it's been over a week with no info on the bz. Looks good to me and we should ping them externally and let them know.